sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClientBuilder.java

```diff
@@ -260,6 +287,7 @@ public TableClientBuilder pipeline(HttpPipeline pipeline) {
      *
      * @return The updated {@link TableClientBuilder}.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores some values in the `Configuration` object. Should we add a note like you did in other methods on this same class?